### PR TITLE
Added method to get the uptime of the server in seconds. 

### PR DESCRIPTION
**Pull request by Seadragon91**
This implements #1650. All other changes are from my atom editor. It trims all lines who are empty, containing spaces/tabs, if that is not okay, I can correct it.

### CODE REVIEW

**Comment by SphinxC0re** [comment]
Looks good to me. Any complains?

File: src/Root.h [hunk 1]
```diff
@@ -70,22 +70,31 @@ class cRoot
 	a_OverworldName should be set for non-overworld dimensions if one wishes that world to link back to an overworld via portals
 	*/
 	cWorld * CreateAndInitializeWorld(const AString & a_WorldName, eDimension a_Dimension = dimOverworld, const AString & a_OverworldName = "");
+
+	/** Returns the up time of the server in seconds */
+	long int GetServerUpTime(void)
+	{
+		return static_cast<long int>(std::chrono::duration_cast<std::chrono::seconds>(std::chrono::steady_clock::now() - m_StartTime).count());
```
Comment (worktycho) [inline src/Root.h]: I'd rather you didn't use long as it causes too many issues with differences between windows and linux, choose either long long, or int.

**Comment by Seadragon91** [comment]
A `int` should be enough. I don't think that someone can run the server with a up time of 68 years :-)

**Comment by madmaxoft** [comment]
I never understood the zealous IDEs that try to remove all the whitespace

**Comment by Seadragon91** [comment]
@madmaxoft Should I disabled the option, that trims the lines of spaces/tabs? It's no problem for me.

**Comment by madmaxoft** [comment]
I'd prefer that, the commits are a mess of useless whitespace changes this way and it's difficult to navigate useful code. Github does have the option to ignore whitespace in the diffs, but then I cannot add comments in such view, so it's pain to use.

**Pull request merged**